RELEASING/release-notes-0-38/README.md

```diff
@@ -0,0 +1,319 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release Notes for Superset 0.38
+
+## What's new
+- [New features](#new-features)
+- [Bugfixes](#bugfixes)
+- [Breaking Changes](#breaking-changes)
+- [Complete Changelog](#complete-changelog)
+
+### New features
+**DATABASES, DATASETS, QUERIES**
+
+[SIP-40] Proposal for Custom Error Messages #9194 ([SIP](https://github.com/apache/incubator-superset/issues/9194))
+- feat: improve presto query perf  (#[11069](https://github.com/apache/incubator-superset/pull/11069))
+- feat: more specific presto error messages  (#[11099](https://github.com/apache/incubator-superset/pull/11099))
+- feat: refactor error components and add database issue code  (#[10473](https://github.com/apache/incubator-superset/pull/10473))
+- feat: welcome presto to the suite of tested databases  (#[10498](https://github.com/apache/incubator-superset/pull/10498))
+
+
+Certification of Data Entities ([Roadmap](https://github.com/apache-superset/superset-roadmap/issues/73))
+- feat: add certification to metrics  (#[10630](https://github.com/apache/incubator-superset/pull/10630))
+- feat: add extra column to tables and sql_metrics  (#[10592](https://github.com/apache/incubator-superset/pull/10592))
+- feat: bump superset-ui for certified tag  (#[10650](https://github.com/apache/incubator-superset/pull/10650))
+
+
+Database CRUD screen refactor/redesign ([Roadmap](https://github.com/apache-superset/superset-roadmap/issues/14))
+- feat: add/edit database modal form sections UI  (#[10745](https://github.com/apache/incubator-superset/pull/10745))
+- feat: base tabbed modal for new database CRUD UI  (#[10668](https://github.com/apache/incubator-superset/pull/10668))
+
+
+Database list view refactor/redesign ([Roadmap](https://github.com/apache-superset/superset-roadmap/issues/55))
+- feat: database delete warning  (#[10800](https://github.com/apache/incubator-superset/pull/10800))
+<img src="media/10800.gif" width="50%"/>
```

Comment:
Could you indent the image three spaces so it's wrapped under the list item?

Reply:
images are now wrapped under the bullet points.